content/events/2021-online-unconference/_index.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Nordic-RSE online unconference"
+description = "Nordic RSE – Research Software Engineers in the nordics – Unconference 29–30.June #NordiRSEunconf"
++++
+
+# Nordic RSE Unconference 2021
+
+We are planning an online unconference on June 29–30. The program will consist of your contributions and we encourage you to submit a short abstract for a discussion topic, talk, demonstration or any other type of program you would like to run.
+
+## Dates: June 29-30, 13-16 CET (3 + 3 hours)
+
+## Format and goal and scope
+
+To bring the community of research software engineers together in 2021,
+to exchange ideas and solutions and challenges we plan an informal unconference.
+In the unconference format we encourage participants to shape the program and
+we will provide support and infrastructure for that.
+
+This event is not only limited to the Nordics and not only limited to participants
+who are or identify as RSEs but anybody interested in the interface of research and software and data is welcome.
+
+
+## What contributions we welcome
+
+In principle we welcome any contributions which respect the [Code of Conduct](/events/2021-online-unconference/code_of_conduct)
+and in principle of any length. We will try to arrange a program based on your input.
+
+Here are some examples of possible contributions:
+- Discussion topics (about software, the research software community, or anything related to RSEs)
+- Demonstrations (tools you like, software you wrote)
+- Quick workshops or reprohacks (only 3 hours in a day)
+- Talks 
+- Workshops and lessons: teach us something new
+- Can also be something unfinished and almost working where you would like to get feedback on
+
+## How to contribute ideas
+
+- Come to the meeting and pitch your discussion idea, talk or workshop.
+- Get support and feedback on your idea before the event by submitting
+  it to as a [GitHub issue](https://github.com/nordic-rse/nordic-rse.github.io/issues/new).
+  <!-- - You can also email us at ... Note that your name and abstract will be published.
+  What is our email?
+  -->
+- Contributions should not violate [Code of Conduct](/events/2021-online-unconference/code_of_conduct)
+
+
+## Tentative format
+
+- Day 1
+  - 13:00 : Pitch topics
+  - 13:10 : Unconference session
+  - 14:20 : Break
+  - 14:40 : Unconference session
+  - 16:00 : Social time
+  - 18:00 : Close
+
+- Day 2
+  - 13:00 : Pitch topics
+  - 13:10 : Unconference session
+  - 14:20 : Break
+  - 14:40 : Unconference session
+  - 16:00 : Social time
+  - 18:00 : Close
+
+
+## What we will provide
+
+- Code of conduct
+- Zoom (with breakout rooms for unconference sessions)
+- HackMD pages
+- Streaming and recording (opt-in)
+- Website (this one)
```

Comment:
Not sure if relevant, but maybe add Zulip channel for communication and discussions before/after (and possibly during) the unconference?